src/utils/raceCountValidator.js

```diff
@@ -0,0 +1,33 @@
+import { Message, ROUND_VALIDATION, INPUT_VALIDATION_REGEX } from './constants.js';
+
+const isNum = (userInput) => {
+  const NUMERIC_REGIX = INPUT_VALIDATION_REGEX.NUMERIC;
+
+  if (NUMERIC_REGIX.test(userInput) === true) {
```

Comment:
조건문을 쓰지않고

```suggestion
return NUMERIC_REGIX.test(userInput)
```
와 같이 바꾸는 건 어떨까요?